### PR TITLE
Passthrough extra args as `string[]`

### PR DESCRIPTION
**Pull request by Jaecen**
Clarg can now handle extra arguments as a `string[]`, making it easy to pass through unparsed args to another system for further parsing.

### CODE REVIEW

File: src/Clarg/Clarg.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net451;netstandard1.5</TargetFrameworks>
```
Comment (duncancoppedge) [inline src/Clarg/Clarg.csproj]: net451 and netstandard1.5 don't look like "latest tooling".  Just making sure we don't want to be at netstandard2.0 for this one.

File: test/Clarg.Test/Clarg.Test.csproj [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
```
Comment (duncancoppedge) [inline test/Clarg.Test/Clarg.Test.csproj]: 1.0?

**Comment by Jaecen** [comment]
This project was in the old 1.0 RTM project.json format. This commit was to move it to the current v2.0 tooling (e.g. .csproj, VS2017 SLN).

File: src/Clarg/Clarg.csproj (hunk 1 above)
Comment (Jaecen) [reply src/Clarg/Clarg.csproj]: This is the targeted output version. Everything here works on older versions of the frameworks, so I left it back there. I could move them forward, but the only thing that would happen is that older client's couldn't update to this version of the library.

File: test/Clarg.Test/Clarg.Test.csproj (hunk 2 above)
Comment (Jaecen) [reply test/Clarg.Test/Clarg.Test.csproj]: `netcoreapp` and `netstandard` are different things and they have different release schedules. This is the equivalent of targeting .Net Framework libraries (2.0, 3.0, 3.5, 4.0, 4.5, 4.6, 4.7) vs. the actual runtimes (just 2.0 and 4.0).

This is correct for this project, as I decided not to move the frameworks forward yet.

**Pull request merged**